docs/src/modules/components/MarkdownDocs.js

```diff
@@ -38,7 +38,7 @@ const styles = theme => ({
   },
 });
 
-const SOURCE_CODE_ROOT_URL = 'https://github.com/mui-org/material-ui/blob/master';
+const SOURCE_CODE_ROOT_URL = 'https://github.com/mui-org/material-ui/blob/next';
```

Comment:
Should we be managing these with a global, or at least creating an umbrella tissue which references all the PRs that change `master` to `next`? We will presumably need to change them back at some point.

Reply:
It's a good point. If I remember correctly for the v1-beta -> v1 release, we have replaced all the `next` matches with `master`.